packages/labs/ssr/src/lib/render-value.ts

```diff
@@ -394,87 +394,82 @@ const getTemplateOpcodes = (result: TemplateResult) => {
             });
           }
         }
-        if (node.attrs.length > 0) {
```

Comment:
Would an alternative fix be lifting the `|| node.isDefinedCustomElement`  check here?

That would only allow through zero length attribute custom element nodes (rather than all nodes). Although I agree with your comment that this filter may be unnecessary as the extra work is on zero length arrays.

Reply:
As discussed offline we consider having less branching more readable when empty arrays will just flow through. Thank you for the discussion!